src/app/components/ATIAnalytics/params/index.test.ts

```diff
@@ -430,29 +383,18 @@ describe('ATIAnalytics params', () => {
           }),
         );
       });
-
-      it('should not invoke buildPageATIUrl for an unmigrated page type with no atiData', () => {
```

Comment:
Do we need another page type to be used and keep this test? What is unmigrated that is suitable here, and do we even need this test anymore?